zmq.go

```diff
@@ -50,6 +62,7 @@ func connFromAddr(addr string) (net.Conn, error) {
 // Conn is a connection to a ZMQ server.
 type Conn struct {
 	conn    net.Conn
+	topics  []string
```

Comment:
I don't see this value ever being set. So it would see that with this, we'd be able to reconnect, but then would lose notifications concerning the topics we care about?

Reply:
Nvm :)

Reply:
See the comment on line 342.